### ISSUE TITLE
Documentation?

### ISSUE THREAD

**Issue by uhlhosting** [nextcloud/talk_matterbridge#36]
It seems that to install the app via Web app was simple, now if someone explain me where to find the rest of steps to get the integration working would be great. 

It seems whoever documented this README.md here does not care about the install procedure, more about the development procedure. Is this in Beta? Should not be used? What am I missing. What is the benefit of installing this and not the Matterbridge itself via cli? Where do configs go in Matterbridge from the web app?

**Comment by gary-kim** [comment]
All the configuration happens in the Nextcloud Talk settings. If you'd like to add that to the README, feel free to make a PR.

**Comment by uhlhosting** [comment]
Well, thats the issue that I see nothing there to configure related to mattermost or matterbridge.

**Comment by sirdrakeistda** [comment]
I searched the config too. Some bridges (e.g WhatsApp) are not configurable in the Talk frontend, so I wanted to add it manually. But I haven't found the config file. Is it possible to add bridges without frontend?

**Comment by gary-kim** [comment]
Feel free to use Matterbridge directly to have full control over the config file. The purpose of the integration with Talk is to let people who are unfamiliar with config files and managing services easily set up bridging between their Nextcloud Talk instances and other chat systems.

**Comment by uhlhosting** [comment]
I agree with you, yet you seem to completely ignore the fact that there are no such options no where inside talk settings nor the plugin settings.

**Comment by gary-kim** [comment]
![image](https://user-images.githubusercontent.com/47195730/109405596-12676080-79ad-11eb-99bd-a8c79d8d9b3f.png)
![image](https://user-images.githubusercontent.com/47195730/109405607-257a3080-79ad-11eb-8e5a-dbb634f7faed.png)

**Comment by sirdrakeistda** [comment]
> Feel free to use Matterbridge directly to have full control over the config file. The purpose of the integration with Talk is to let people who are unfamiliar with config files and managing services easily set up bridging between their Nextcloud Talk instances and other chat systems.
Where do I find the config files of Matterbridge to configure them manually. I haven't found them. I looked in every ".toml" file on my NC and I looked also in the spreed and talk_matterbridge folder. But I haven't found any config except for temporary files in /tmp.

**Comment by gary-kim** [comment]
To use Matterbridge directly, refer to the Matterbridge documentation: https://github.com/42wim/matterbridge#running and https://github.com/42wim/matterbridge/wiki/How-to-create-your-config

Nextcloud Talk support was added to Matterbridge recently so that can be used: https://github.com/42wim/matterbridge/wiki/Section-Nextcloud-Talk-%28basic%29

**Comment by gary-kim** [comment]
All in all, let's keep this issue open. If someone wants to make a PR that adds more detailed documentation to the README to avoid further misunderstanding, that would be very much appreciated.

**Comment by Mannshoch** [comment]
I read links but also still do not understand where to set up this toml files. Could someone tell me where to place it in Nextcloud?

**Comment by Mannshoch** [comment]
I searched again now I finaly found a blog that. Said how to configure Matterbridge. I newer had the Idea to look into Talk in the right Panel.  Sorry for this misunderstanding.

**Comment by Mannshoch** [comment]
Hm.. Seems I missunderstood more. I assumed I could set-up my accounts in my profile and automatic get all teams/channels I have eg. on Mattermost with a representative into talk and may get access to the contact list. 
Not sure if I could use Matterbridge that way.

**Comment by nickvergessen** [comment]
I guess it makes sense to put it there, yes.
And then link to it from the Readme of this repo and description of the app in info.xml?

**Comment by Routhinator** [comment]
Be aware that none of these options appear in the Android or Desktop clients. Options from all other integrations show, but Matterbridge does not.

Desktop Client:
<img width="930" height="963" alt="Image" src="https://github.com/user-attachments/assets/72ce1fae-2fee-4a77-b7d0-6d25596f1f15" />


Web Client:

<img width="921" height="879" alt="Image" src="https://github.com/user-attachments/assets/224b367f-f19f-4829-b47c-d2a95a28ca20" />

**Comment by WilliamDEdwards** [comment]
For anyone else coming across this:

There is _some_ documentation on https://docs.nextcloud.com/server/latest/user_manual/lt/talk/matterbridge.html, which says that Matterbridge itself is shipped with the app as a binary.

And https://nextcloud-talk.readthedocs.io/en/latest/matterbridge/ says:

> When this app is used, the basic configuration should work out of the box.

So, no manual configuration / system setup needs to be done, apparently.

So:

1. Install Matterbridge app in Nextcloud
2. Go to `/settings/admin/talk` and enable Matterbridge there (bottom of the page)
3. 'Matterbridge' appears as an option in conversation settings

However, the bridge dropdown says Matterbridge isn't running, and this is in the Nextcloud log:

```
{"reqId":"5VzPECYJXV5tHFfxEyLd","level":3,"time":"2026-05-02T16:32:10+00:00","remoteAddr":"2a0c:eb00:0:f7:185:233:175:161","user":"30F2A4F5-C686-458C-9A61-9F533B98E8DF","app":"no app in context","method":"PUT","url":"/ocs/v2.php/apps/spreed/api/v1/bridge/yjccihsh","scriptName":"/ocs/v2.php","message":"preg_match(): Argument #2 ($subject) must be of type string, false given in file '/var/www/nextcloud/apps/spreed/lib/MatterbridgeManager.php' line 505","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:150.0) Gecko/20100101 Firefox/150.0","version":"32.0.5.0","exception":{"Exception":"Exception","Message":"preg_match(): Argument #2 ($subject) must be of type string, false given in file '/var/www/nextcloud/apps/spreed/lib/MatterbridgeManager.php' line 505","Code":0,"Trace":[{"file":"/var/www/nextcloud/lib/private/AppFramework/App.php","line":153,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->"},{"file":"/var/www/nextcloud/lib/private/Route/Router.php","line":321,"function":"main","class":"OC\\AppFramework\\App","type":"::"},{"file":"/var/www/nextcloud/ocs/v1.php","line":61,"function":"match","class":"OC\\Route\\Router","type":"->"},{"file":"/var/www/nextcloud/ocs/v2.php","line":8,"args":["/var/www/nextcloud/ocs/v1.php"],"function":"require_once"}],"File":"/var/www/nextcloud/lib/private/AppFramework/Http/Dispatcher.php","Line":150,"Previous":{"Exception":"TypeError","Message":"preg_match(): Argument #2 ($subject) must be of type string, false given","Code":0,"Trace":[{"file":"/var/www/nextcloud/apps/spreed/lib/MatterbridgeManager.php","line":505,"function":"preg_match"},{"file":"/var/www/nextcloud/apps/spreed/lib/MatterbridgeManager.php","line":114,"function":"validateParts","class":"OCA\\Talk\\MatterbridgeManager","type":"->"},{"file":"/var/www/nextcloud/apps/spreed/lib/Controller/MatterbridgeController.php","line":85,"function":"editBridgeOfRoom","class":"OCA\\Talk\\MatterbridgeManager","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/Http/Dispatcher.php","line":204,"function":"editBridgeOfRoom","class":"OCA\\Talk\\Controller\\MatterbridgeController","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/Http/Dispatcher.php","line":118,"function":"executeController","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/App.php","line":153,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->"},{"file":"/var/www/nextcloud/lib/private/Route/Router.php","line":321,"function":"main","class":"OC\\AppFramework\\App","type":"::"},{"file":"/var/www/nextcloud/ocs/v1.php","line":61,"function":"match","class":"OC\\Route\\Router","type":"->"},{"file":"/var/www/nextcloud/ocs/v2.php","line":8,"args":["/var/www/nextcloud/ocs/v1.php"],"function":"require_once"}],"File":"/var/www/nextcloud/apps/spreed/lib/MatterbridgeManager.php","Line":505},"message":"preg_match(): Argument #2 ($subject) must be of type string, false given in file '/var/www/nextcloud/apps/spreed/lib/MatterbridgeManager.php' line 505","exception":{},"CustomMessage":"preg_match(): Argument #2 ($subject) must be of type string, false given in file '/var/www/nextcloud/apps/spreed/lib/MatterbridgeManager.php' line 505"}}
```

That said, setting up new Slack bots is impossible (https://github.com/42wim/matterbridge/issues/2159), and this app is yet to migrate to the non-dead fork (https://github.com/nextcloud/talk_matterbridge/issues/121).
